applications/ecallmgr/src/ecallmgr_app.erl

```diff
@@ -72,3 +76,17 @@ declare_exchanges() ->
 node_bindings() ->
     _ = kz_nodes_bindings:bind('ecallmgr', ?MODULE),
     'ok'.
+
+-spec freeswitch_nodesup_bind() -> 'ok'.
+freeswitch_nodesup_bind() ->
+    _ = kazoo_bindings:bind(<<"freeswitch.node.modules">>, ?MODULE, 'freeswitch_node_modules'),
+    'ok'.
+
+-spec freeswitch_nodesup_unbind() -> 'ok'.
+freeswitch_nodesup_unbind() ->
+    _ = kazoo_bindings:unbind(<<"freeswitch.node.modules">>, ?MODULE, 'freeswitch_node_modules'),
+    'ok'.
+
+-spec freeswitch_node_modules() -> ne_binaries().
+freeswitch_node_modules() ->
+    application:get_env(?APP, 'node_modules', ?NODE_MODULES).
```

Comment:
@fenollp not getting what you're saying. `?NODE_MODULES` is defined once in ecallmgr.hrl and referred once in here.